void-gr-filters.txt

```diff
@@ -177,6 +178,7 @@ zougla.gr/uploads/thomas/travel_city_banner*
 ||cdn1.bbend.net/media/com_news/ads/*
 ||cdnprojectagora.s3.amazonaws.com/gr/gazzetta.js$script,domain=gazzetta.gr
 ||cheapis.gr/widget/*
+||clevernt.com^
```

Comment:
this is included in other blocklists such as Peter Lowe's adservers. Check it here: https://pgl.yoyo.org/adservers/

No need to include it in the Greek adblock list, as it's not only related to Greek sites.

Reply:
Fair enough, it was included in a non-standard list that I didn't know of, thanks for bringing it to my attention